integration_tests/mqtt/create_source.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE
+  personnel (id integer, name varchar);
+
+CREATE TABLE mqtt_source_table
+(
+  id integer,
+  name varchar,
+)
+WITH (
+    connector='mqtt',
+    url='tcp://mqtt-server',
+    topic= 'test',
+    qos = 'at_least_once',
+) FORMAT PLAIN ENCODE JSON;
+
+
+CREATE SINK mqtt_sink
+FROM
+  personnel
+WITH
+  (
+    connector='mqtt',
+    url='tcp://mqtt-server',
+    topic= 'test',
```

Comment:
The output topic is the same as the input topic, forming a loop. Is this intended?

It looks a bit weird to me. Recommend using separated topics for input and output, and writing a `sink_check.py` for outputted data. Please search for `sink_check.py` for examples.

Reply:
I took this from the NATS integration tests, it does this `table -> sink -> mqtt broker -> source` , so every record inserted in the table ends up in the source table using MQTT, but I will add a sink_check